shell/platform/darwin/common/framework/Source/FlutterCodecs.mm

```diff
@@ -73,7 +73,7 @@ - (NSData*)encode:(id)message {
 }
 
 - (id)decode:(NSData*)message {
-  if (message == nil)
+  if (message == nil || message.length == 0)
```

Comment:
You cal get rid of the `message==nil` check. `message.length` sends an Objective C message of `length` to `message`. Messages to `nil` objects return zero.

Reply:
Done